.pre-commit-config.yaml

```
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3
+    rev: v4.0.0-alpha.8
```

I've not looked into why prettier is failing, but perhaps it's better to skip the alphas:

```suggestion
    rev: v3.1.0
```